ui/StatusQ/src/StatusQ/Popups/StatusMenu.qml

```diff
@@ -88,6 +88,11 @@ Menu {
     delegate: StatusMenuItem {
         visible: root.hideDisabledItems ? enabled : true
         height: visible ? implicitHeight : 0
+        onImplicitWidthChanged: {
+            var adjustedImplicitWidth = implicitWidth + 2 * padding
```

Comment:
I'm not sure, but is this part even needed?

`Menu` is a `Popup`. And it's `contentWidth` should be calculated automatically based on `contentItem` implicit width (which is a `StatusListView` in our case).

And I suppose that `StatusListView` should calculate it's contentWidth from `implicitWidth` of all items 🤔 

So the real change is the one you did in `StatusMenuItem`, but it wasn't enough?